pkg/server/container_execsync.go

```diff
@@ -100,16 +149,11 @@ func (c *criContainerdService) ExecSync(ctx context.Context, r *runtime.ExecSync
 	if err != nil {
 		return nil, fmt.Errorf("failed to wait for exec in container %q to finish: %v", id, err)
```

Comment:
Maybe we should let `waitContainerExec` also return pointer, so that we don't need the `unknownExitStatus` for now.

We don't use the exit code when there is an error now.